core/src/main/java/io/seata/core/rpc/netty/NettyClientChannelManager.java

```diff
@@ -181,13 +187,29 @@ void reconnect(String transactionServiceGroup) {
         }
         Set<String> channelAddress = new HashSet<>(availList.size());
         try {
+            Map<String, Exception> failedMap = new HashMap<>();
             for (String serverAddress : availList) {
                 try {
                     acquireChannel(serverAddress);
                     channelAddress.add(serverAddress);
                 } catch (Exception e) {
-                    LOGGER.error("{} can not connect to {} cause:{}", FrameworkErrorCode.NetConnect.getErrCode(),
-                        serverAddress, e.getMessage(), e);
+                    failedMap.put(serverAddress, e);
+                }
+            }
+            if (failedMap.size() > 0) {
+                if (LOGGER.isInfoEnabled()) {
+                    LOGGER.error("{} can not connect to {} cause:{}",
+                            FrameworkErrorCode.NetConnect.getErrCode(), failedMap.keySet(),
+                            failedMap.values().stream().map(Throwable::getMessage).collect(Collectors.toSet()));
+                } else if (LOGGER.isDebugEnabled()) {
+                    failedMap.forEach((key, value) -> {
+                        LOGGER.error("{} can not connect to {} cause:{} trace information:{}",
+                                FrameworkErrorCode.NetConnect.getErrCode(), key, value.getMessage(), value);
+                    });
+                }
+                String invalidAddress = StringUtils.join(failedMap.keySet().iterator(), ", ");
```

Comment:
Implementation problems.failFast is only used for startup logic and should not be included in public processing logic.